cmd/habitat/community/manager.go

```diff
@@ -239,6 +248,7 @@ func (m *Manager) addCommunity(communityID string, communityState *state.Communi
 	return nil
 }
 
+//nolint
```

Comment:
why no lint?

Reply:
this function is unused which the linter doesn't like, but I don't want to get rid of it